x-pack/solutions/security/plugins/security_solution/public/plugin.tsx

```diff
@@ -436,7 +442,10 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
     const { upsellingService, isSolutionNavigationEnabled$ } = this.contract;
 
     // When the user does not have access to SIEM (main Security feature) nor Security Cases feature, the plugin must be inaccessible.
-    if (!capabilities.siem?.show && !capabilities.securitySolutionCasesV2?.read_cases) {
+    if (
+      !capabilities[SECURITY_FEATURE_ID]?.show &&
```

Comment:
Why not use the helper here?

Reply:
Nice catch, I added it here https://github.com/elastic/kibana/pull/201780/commits/3482cec105d482616c3f92b757fde62269bbc57c